addon/services/scroll-activity.js

```diff
@@ -19,6 +19,7 @@ export default Service.extend(Evented, {
   init() {
     this._super(...arguments);
 
+	if (!self.document) { return; }
```

Comment:
I'm not sure why we would want to skip all setup if there is no document. Does `init` get called a second time after fastboot boots the app?